docs/source/setup.md

```diff
@@ -202,15 +202,15 @@ npm install -g bower
 
 ### Step 4: Start the development environment
 
-* To run backend development server at [backend]
+* To run backend development server at [backend]:
 
 ```
 # activate virtual environment if not activated
 source venv/bin/activate
 python manage.py runserver --settings=settings.dev
 ```
 
-* To run frontend development server for at [frontend]
+* To run frontend development server for at [frontend]:
```

Comment:
Please specify the port number as `127.0.0.1:8888` here as done in above command.